# Generated by Django 5.1.4 on 2025-06-19 00:03

import django.utils.timezone
from django.conf import settings
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('AppHome', '0010_demanda_data_verificacao'),
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='TelegramLog',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('data_envio', models.DateTimeField(default=django.utils.timezone.now)),
                ('destinatario', models.CharField(max_length=100)),
                ('mensagem', models.TextField()),
                ('sucesso', models.BooleanField(default=True)),
                ('erro', models.TextField(blank=True, null=True)),
            ],
        ),
        migrations.AddField(
            model_name='cliente',
            name='usuarios_responsaveis',
            field=models.ManyToManyField(blank=True, related_name='clientes_responsaveis', to=settings.AUTH_USER_MODEL, verbose_name='Usuários Responsáveis'),
        ),
    ]
